src/clients/extension-manager.js

```diff
@@ -55,6 +55,11 @@ export async function getExtension(canonicalName) {
   return await jsonApi.get(url);
 }
 
+export async function getPlatform(id) {
+  const url = extensionManagerUri.clone().segment(`/v1/platforms/${id}`);
+  return await jsonApi.get(url);
+}
+
```

Comment:
If this is an async function that uses await only in the final return statement, isn't that the same thing as a normal function without async/await?

Reply:
It pretty much is, I've used this because this file is full of such methods, and I did not want to cause any confusion by having be the sole exception.

We definitely should look into that thing as a whole in future improvements attempts in the cli.

Reply:
I'd just like to add that I believe we should not have an await in a return statement because it's harder to debug. We should save the return value of the await into a const, and then return that const.